setup1-contributor/src/commands/contribute.rs

```diff
@@ -934,11 +935,29 @@ fn update_progress_bar(updater: StatusUpdater, progress_bar: ProgressBar) -> tok
 /// Utility structure to provide updates about the round progress
 struct StatusUpdater {
     server_url: Url,
+    private_key: PrivateKey<Testnet2Parameters>,
     participant_id: String,
 }
 
 impl StatusUpdater {
     async fn status_updater(&self, progress_bar: ProgressBar) -> Result<()> {
+        let status = get_contributor_status(&self.server_url, &self.private_key).await?;
+        match status {
+            ContributorStatus::Queue => {
+                progress_bar.set_message(&"In queue for one of the future rounds");
+            }
+            ContributorStatus::Round => {
+                self.update_position_in_round(&progress_bar).await?;
+            }
+            ContributorStatus::Other => {
+                progress_bar.set_message(&"Neither in queue nor in the current round");
```

Comment:
This should probably be an error message, since we should never see the situation where you are not in the queue or in the round.

The text should also be: "Not in the queue for Aleo Setup ceremony. Please double check the address you are connecting, then disconnect and try again"

Reply:
Adding more errors to the logs doesn't help, because progress bar runs in a separate loop, and it doesn't affect the main loop of the contributor. But adding a good message helps to figure out what's going on.